borg/testsuite/helpers.py

```diff
@@ -399,3 +402,44 @@ def test_get_cache_dir():
     # reset old env
     if old_env is not None:
         os.environ['BORG_CACHE_DIR'] = old_env
+
+@pytest.fixture()
+def stats():
+    stats = Statistics()
+    stats.update(10, 10, unique=True)
```

Comment:
if you used different values for size and csize, some more error conditions could be discovered.